src/pages/AdminPage/CircleComponent.tsx

```diff
@@ -0,0 +1,235 @@
+import { GearIcon, InfoCircledIcon } from '@radix-ui/react-icons';
```

Comment:
Thanks for breaking things out into a separate file. Since this file doesn't contain a component named CircleComponent, please rename it to something like `components`, `helpers`, etc.